### PR TITLE
updated limits to reflect v5 version

### PR DESCRIPTION
**Pull request by caitlindavey**
## Description

Updated limits page to reflect new v5 PAYG pricing based on support ticket

* [EDU-1174](https://ably.atlassian.net/browse/EDU-1174)

## Review

Please ensure all the correct information has been updated based on https://ably.atlassian.net/wiki/spaces/product/pages/2447179818/PDR+50+Accommodate+higher+limits+in+v5+PAYG

@m-hulbert and pass to Mike Clark for final review 

[EDU-1174]: https://ably.atlassian.net/browse/EDU-1174?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/general/limits.textile [hunk 1]
```diff
@@ -78,8 +78,8 @@ The quotas for an account can be considered a soft limit, with no restrictions c
 
 |_/2=. Limit |_\2=. Free |_\2=. PAYG |_\2=. Committed Use |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
-| "Peak connections":#peak-connections | 200 | 240 | 10,000 | 25,000 |\2. Custom >= PAYG |
-| "Peak channels":#peak-channels | 200 | 240 | 10,000 | 25,000 |\2. Custom >= PAYG |
+| "Peak connections":#peak-connections | 200 | 240 | 10,000 | 250,000 |\2. Custom >= PAYG |
```
Comment (m-hulbert) [inline content/general/limits.textile:81]: I think the soft limit for peak connections should also be 100,000 here.

**Review by m-hulbert — CHANGES_REQUESTED** [review]
I can't see the changes for the below rates in addition to the one I've commented on: 

* connections.maxCreationRate
* channels.maxCreationRate
* messages.instantaneous.hard.maxRate
* API requests maxRate

File: content/general/limits.textile [hunk 2]
```diff
@@ -78,11 +78,11 @@ The quotas for an account can be considered a soft limit, with no restrictions c
 
 |_/2=. Limit |_\2=. Free |_\2=. PAYG |_\2=. Committed Use |
 |_. Soft |_. Hard |_. Soft |_. Hard |_. Soft |_. Hard |
-| "Peak connections":#peak-connections | 200 | 240 | 10,000 | 250,000 |\2. Custom >= PAYG |
+| "Peak connections":#peak-connections | 200 | 240 | 100,000 | 250,000 |\2. Custom >= PAYG |
 | "Peak channels":#peak-channels | 200 | 240 | 100,000 | 250,000 |\2. Custom >= PAYG |
 | "Total monthly messages":#messages | 6,000,000 | 7,200,000 | No limit | No limit |\2. No limit |
 | "Messages":#messages (per hour) | 84,000 | 100,000 | 1,400,000 | 3,500,000 |\2. Custom >= PAYG |
-| "Message rate":#messages (per second) | - | 70 | - | 2,400 |\2. Custom >= PAYG |
+| "Message rate":#messages (per second) | - | 70 | - | 4,000 |\2. Custom >= PAYG |
```
Comment (m-hulbert) [inline content/general/limits.textile:85]: This one should be 25,000.

File: content/general/limits.textile (hunk 1 above)
Comment (caitlindavey) [reply content/general/limits.textile]: I believe I updated this! Whew

**Review by m-hulbert — COMMENTED** [review]
Just missing the change to `API requests maxRate` now.

**Comment by caitlindavey** [comment]
Can you drop in what we call that in the doc? I'm not sure if it's API requests (per hour), API requests (per second), or "Authenticated access token requests":#authenticated-token (per hour). Sorry and thank you for your help!

**Review by m-hulbert — COMMENTED** [review]
It's API requests (per second).

**Comment by caitlindavey** [comment]
Updated!

**Review by m-hulbert — APPROVED** [review]

**Pull request merged**